superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -485,7 +485,7 @@ export const ControlPanelsContainer = (props: ControlPanelsContainerProps) => {
         ? baseDescription(exploreState, controls[name], chart)
         : baseDescription;
 
-    if (name === 'adhoc_filters') {
+    if (name.startsWith('adhoc_filters')) {
```

Comment:
is it guaranteed that the filter starts with this? why not using `includes` for example?